### PR TITLE
added new sections in the cycle count

### PR DESCRIPTION
**Pull request by Shreyaakothari**

### CODE REVIEW

File: netsuite/Flows/cycle-count.md [hunk 1]
```diff
@@ -10,19 +10,24 @@ To support this process, HotWax Commerce offers the Cycle Count app, which empow
...
 
 ### Pushing Cycle Count File to HotWax Commerce
-Store associates upload the Cycle Count file from the app to the HotWax Commerce server. Store managers review this file, approve it, and subsequently, inventory variances are recorded in HotWax Commerce.
-
+Store associates upload the Cycle Count file from the app to the HotWax Commerce server. Store managers review this file, approve it, and subsequently, inventory cycle count variances are recorded in HotWax Commerce.
 
 ### Export Inventory Variances from HotWax Commerce
-Once inventory variances are successfully logged in HotWax Commerce, a specific job is executed to create a CSV file containing all the inventory variance records. This file is then placed in an SFTP location, ensuring that it is readily accessible for further processing.
+Once inventory cycle count variances are successfully logged in HotWax Commerce, a specific job is executed to create a CSV file containing all the cycle count inventory variance records. This file is then placed in an SFTP location, ensuring that it is readily accessible for further processing.
```
Comment (dt2patel) [inline netsuite/Flows/cycle-count.md]: Add where this jobs runs. For example jobs runs in the HotWax integration platform

**Review by dt2patel — CHANGES_REQUESTED** [review]
Small change requested

**Comment by Shreyaakothari** [comment]
> Small change requested

Updated. Please check

File: netsuite/Flows/cycle-count.md (hunk 1 above)
Comment (Shreyaakothari) [reply netsuite/Flows/cycle-count.md]: Updated

**Pull request merged**